guidance/03-analysis/analysis-steps/02-analysis-modelling.qmd

```diff
@@ -7,8 +7,64 @@ image-alt: Photo by Shubham Dhage on Unsplash
 
-## Conducting the work
-Starting from the source tables, we want to create the agreed outputs. Depending on the work, there are many approaches and tools we can use to do this.
+## Selection of methods
+
+The analysis first requires an appropriate method that can provide the desired outputs.
+
+*	Create a short-list of methods that could be applicable to the problem.
+*	Consider the pros and cons of each candidate method:
+  *	are there examples of this method being applied to the problem archetype?
+  *	will the available data support the method? If not, is it feasible to transform the data into a more suitable form?
+  *	will the tools available support the method?
+*	Decide which methods are best suited to the problem:
+  *	often the best method will be clear
+  *	sometimes you may need to try a small number of methods before deciding on the overall best one
+
+Keep notes of your decision-making and method trials for the final write-up of the initiative.
+
+## Application of methods
+
+Apply the selected model(s) with **RIGOUR**; it should be **R**epeatable, **I**ndependent, **G**rounded in reality, **O**bjective, **U**ncertainty-managed, and **R**obust.
+
+* **Repeatable**: For an analytical process to be considered ‘valid’ it might reasonably be expected that for the “same” inputs and constraints the analysis produces the “same” outputs. It is important to note that different analysts will consider the analytical problem differently, potentially resulting in differing results, however if any one approach is repeated the results should be as expected.
+* **Independent**: To produce analysis that is free of prejudice or bias. In doing so, care should be taken to appropriately balance the views across all stakeholders and experts.
+* **Grounded in reality**: Quality analysis takes the commissioner and analyst on a journey as views and perceptions are challenged and connections are made between the analysis and its real consequences. Connecting with reality in this way guards against failing to properly grasp the context of the problem which is being analysed.
+* **Objective**: Effective engagement and suitable challenge reduces potential bias and enables the commissioner and the analyst to be clear about the interpretation of the analytical results.
+*	**Uncertainty-managed**: Uncertainties have been identified, managed, and communicated throughout the analytical process.
+*	**Robust**: Provide the analytical result in the context of residual uncertainty and limitations to ensure it is used appropriately.
+
+## Quality Assurance (QA)
+
+Check your analysis thoroughly and often, not as a last step before delivery.
+
+*	Documentation is important because:
+  *	it allows us to transfer knowledge about the analysis
+  *	it keeps the analysis transparent
+  *	it provides instructions on how to use any interactive output
+*	Good analysis structure helps:
```

Comment:
Again not sure it needs a bullet point. Not seen this done anywhere else in the document